solutions/15.js

```diff
@@ -0,0 +1,11 @@
+const solution = (str, char, i = 0) => {
+  if (i < str.length) {
+    if (str[i] == char) {
+      return true;
+    }
+    return solution(str, char, i + 1);
+  }
+  return false;
+};
```

Comment:
In this case, I do believe that putting things outside if statements looks better:
```
if(i>=str.length) return false;
if(str[i] == char) return true;
return solution(str,char, i+1);
```